packages/uows-client-generator/src/index.ts

```diff
@@ -1,63 +1,60 @@
-import * as soap from "soap";
-import * as fs from "fs";
+#!/usr/bin/env node
 
-const wsdlUrl: string = 'https://api.facilities.rl.ac.uk/ws/UserOfficeWebService?wsdl';
-const filePath: string = './UOWSSoapInterface.ts';
```

Comment:
Could this be restored and renamed to `defaultFilePath`, and the actual file path be chosen in a similar way to the one I suggested for `defaultWsdl`? I think having these values at the top of the file makes it easier to see what's going on